src/oscar/apps/dashboard/apps.py

```diff
@@ -16,40 +15,31 @@ class DashboardConfig(OscarDashboardConfig):
         'index': (['is_staff'], ['partner.dashboard_access']),
     }
 
+    def get_app_label_and_url_endpoint_mappings(self):
+        return {
+            'catalogue_dashboard': 'catalogue/',
+            'reports_dashboard': 'reports/',
+            'orders_dashboard': 'orders/',
+            'users_dashboard': 'users/',
+            'pages_dashboard': 'pages/',
+            'partners_dashboard': 'partners/',
+            'offers_dashboard': 'offers/',
+            'ranges_dashboard': 'ranges/',
+            'reviews_dashboard': 'reviews/',
+            'vouchers_dashboard': 'vouchers/',
+            'communications_dashboard': 'comms/',
+            'shipping_dashboard': 'shipping/',
+        }
+
```

Comment:
While you are reworking how the url prefixes and such are done, would you consider making them translatable?

See https://docs.djangoproject.com/en/3.1/topics/i18n/translation/#translating-url-patterns

It can be done like this:
```
return {
             'catalogue_dashboard': _('catalogue/'),
             'reports_dashboard': _('reports/'),
             'orders_dashboard': _('orders/'),
             'users_dashboard': _('users/'),
             'pages_dashboard': _('pages/'),
             'partners_dashboard': _('partners/'),
             'offers_dashboard': _('offers/'),
             'ranges_dashboard': _('ranges/'),
             'reviews_dashboard': _('reviews/'),
             'vouchers_dashboard': _('vouchers/'),
             'communications_dashboard': _('comms/'),
             'shipping_dashboard': _('shipping/'),
         }
```
When composing the full urls, you can make use of ```django.utils.text.format_lazy``` to create the *translatable* view patterns.

Reply:
@specialunderwear this is a good idea - I'm going to request that we handle it in a separate PR though, just so that this PR can focus on the iffeature change which is quite substantial in itself.

Reply:
(The translation change looks like it will be quite a simple diff after this).

Reply:
I think so yes, also it is not important for the dashboard, but more important for the frontend really.

Reply:
It is worthwile to keep the translatable urls in mind while completing this though.